Tools/machines/lxplus-cern/spack.yaml

```diff
@@ -0,0 +1,47 @@
+spack:
+# CUDA
+  definitions:
+  - cuda: []
+  - cuda: [cuda ^ncurses+symlinks, blaspp +cuda ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+# basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python ^ncurses+symlinks, py-pip ^ncurses+symlinks, py-mpi4py ^ncurses+symlinks, py-numpy ^ncurses+symlinks,
+      py-periodictable ^ncurses+symlinks, py-picmistandard ^ncurses+symlinks]
+    when: env.get("SPACK_STACK_USE_PYTHON", "") == "1"
+  specs:
+  - ncurses+symlinks
+  - adios2 ^ncurses+symlinks
+  - blaspp ^ncurses+symlinks
+  - ccache ^ncurses+symlinks
+  - cmake ^ncurses+symlinks
+  - fftw ^ncurses+symlinks
+  - hdf5 ^ncurses+symlinks
+  - lapackpp ^ncurses+symlinks
+  - mpi ^ncurses+symlinks
+  - openpmd-api ^ncurses+symlinks
+  - pkgconfig
+  - $cuda
+  - $python
+
+  packages:
+    all:
+      target: [x86_64]
```

Comment:
This is usually the CPU sub-architecture and can be tuned if LXPLUS is not a heterogeneous cluster.
Is there more than one kind of CPU on this machine?

Reply:
That's generic and OK for now, would only be relevant if we want to tickle out a bit more performance on pure-CPU runs. We can add a comment with `#` above the line to remind us of that if we ever need to.

Reply:
Comment added :)